app/services/onlinetesting/Phase3TestService.scala

```diff
@@ -267,6 +270,8 @@ trait Phase3TestService extends OnlineTestService with Phase3TestConcern {
       for {
         testGroup <- phase3TestRepo.getTestGroupByToken(launchpadInviteId)
         _ <- phase3TestRepo.updateProgressStatus(testGroup.applicationId, ProgressStatuses.PHASE3_TESTS_RESULTS_RECEIVED)
+        // Launchpad only: If results have been sent for a user, unexpire them
+        _ <- removeExpiryStatus(testGroup.applicationId)
```

Comment:
I don't know whether i'd rather the user ended up in results_received but expired, or got rolled back to started without any evidence that they were in expired to begin with, but with an expiry date in the past - seemed like a toss up